### DISCUSSION TITLE
Fees structure

### DISCUSSION THREAD

**Discussion by jltari** [gochain-io/docs#116] [Q&A]
Hello all,

I need to understand what the fees structure will be over time.
Today, tx are extremely cheap, but how will that evolve ? I suppose there is some plan...

**Comment by treeder** [comment]
It really depends on the price of GO. Although, there's some general agreement that if the price goes up too much, we'll do another vote like this https://github.com/gochain/gips/issues/29, to decrease it by 10x. We'd like the cost of a basic transaction to stay under a penny.

**Reply by jltari** [reply]
OK, Thanks.
I got a copy of the white paper and will carefully read it to understand the process.
